### PR TITLE
Link out to developer documentation for testing notes

### PR DESCRIPTION
**Pull request by zachmargolis**
The extra description is for searchability/SEO

### CODE REVIEW

**Review by amathews-fs — APPROVED** [review]
LGTM

**Review by aduth — APPROVED** [review]

**Review by orenyk — APPROVED** [review]
looks good!

File: _articles/identity-proofing-testing.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Instead of uploading images of IDs (since we aren't supposed to have PII in sandbox environments),
+you can upload a [specially-formatted YML file](https://developers.login.gov/testing/#data-testing).
+
+## Sample SSN
+
+Our developer docs have [sample SSNs](https://developers.login.gov/testing/#personal-information-verification)
+to get desired behavior.
+
+## Sample Phone Numberes
```
Comment (aduth) [inline _articles/identity-proofing-testing.md:44]:
```suggestion
## Sample Phone Numbers
```

**Review by aduth — APPROVED** [review]
Nice 👍

File: _articles/identity-proofing-testing.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/identity-proofing-testing.md:44]: 🤦 thanks for the catch

**Pull request merged**